cmd/keeper/cmd/keeper.go

```diff
@@ -116,11 +116,47 @@ type config struct {
 	pgSUPasswordFile        string
 	pgInitialSUUsername     string
 	pgInitialSUPasswordFile string
+	pgOverrideParameters    []string
 
 	neverMaster             bool
 	neverSynchronousReplica bool
 }
 
+// This function parses the catpured `--postgresql-parameter` flags that are
+// used to override PostgreSQL configurations for a stolon node. The flag
+// `--postgresql-cluster` can be passed multiple times providing the ability to
+// have multipe overrides, thus it is a stringSlice.
+//
+// The usge of the flag is in the following format:
+//
+//  ```
+//  /bin/stolon-keeper --postgresql-parameter key1=value1 \
+//      --postgresql-parameter key2=value2
+//  ```
+//
+// Where key1 and key2 are the postgresql parameters to be overrident, and
+// value1 and value2 are their respective override values.
+//
+// The function loops over the parameters captures, and splits them by the first
```

Comment:
I don't think we need this part of the comment up until the function declaration. The first half of the comment is great, as it explains why we have this and what it is for. From this line on, you describe what the code does, which isn't ideal as now you've duplicated your logic.

I would keep the first half and drop everything from this line!